building/devices/index.rst

```diff
@@ -14,4 +14,5 @@ Device specific information
     qemu
     rpi3
     ti
+    zynqmp
```

Comment:
I would like to move this up between `qemu` and `rpi3`, since the menubar have listed all devices in alphabetic order. Since the title in the document is "`QEMU for ZynqMP zcu102` that would make it follow the same pattern (see the [rendered output](https://optee.readthedocs.io/building/gits/optee_docs.html#build-optee-docs) after applying your patch and you will understand what I mean).